### PR TITLE
Field used for label does not exist

### PR DESCRIPTION
**Pull request by psihius**
Change 'choice_label' to use a closure that renders customers full name and e-mail.
Nor field `name`, nor method `getName` exist in Customer entity
Should go to 1.0, 1.1 and master

| Q               | A
| --------------- | -----
| Branch?         | 1.0
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | None
| License         | MIT

### CODE REVIEW

**Comment by pamil** [comment]
Could you rebase it to `1.0`?

File: src/Sylius/Bundle/CustomerBundle/Form/Type/CustomerChoiceType.php [hunk 1]
```diff
@@ -56,7 +56,9 @@ public function configureOptions(OptionsResolver $resolver): void
                 return $this->customerRepository->findAll();
             },
             'choice_value' => 'email',
-            'choice_label' => 'name',
+            'choice_label' => function (Customer $customer) {
```
Comment (lchrusciel) [inline src/Sylius/Bundle/CustomerBundle/Form/Type/CustomerChoiceType.php]: We should typehint to CustomerInterface

Comment (pamil) [reply src/Sylius/Bundle/CustomerBundle/Form/Type/CustomerChoiceType.php]: Fixed.

**Comment by pamil** [comment]
Thank you, Arvids! 🎉

**Pull request merged**